src/Command/PHPUnitCommand.php

```diff
@@ -74,6 +78,11 @@ private function resolveOptions(InputInterface $input)
         if ($this->supportsCoverage() && $input->getOption('coverage-clover')) {
             $options[] = sprintf('--coverage-clover %s/coverage.xml', getcwd());
         }
+        foreach (['fail-on-warning', 'fail-on-risky', 'fail-on-skipped', 'fail-on-warning'] as $option) {
+            if ($input->getOption($option)) {
+                $options[] = '--'.$option;
```

Comment:
I bet this is the php-cs-fixer problem that is happening @ travis and GHA. This project doesn't use spaces around the concat operator. Sharing because it's tricky to find and it has happened to me in the past. ;-)

Reply:
Ah, you found it! LOL. Just refreshed!